rest_framework/fields.py

```diff
@@ -906,10 +906,10 @@ def __init__(self, max_digits, decimal_places, coerce_to_string=None, max_value=
         super(DecimalField, self).__init__(**kwargs)
 
         if self.max_value is not None:
-            message = self.error_messages['max_value'].format(max_value=self.max_value)
+            message = lazy(self.error_messages['max_value'].format, str)(max_value=self.max_value)
             self.validators.append(MaxValueValidator(self.max_value, message=message))
         if self.min_value is not None:
-            message = self.error_messages['min_value'].format(min_value=self.min_value)
+            message = lazy(self.error_messages['min_value'].format, str)(min_value=self.min_value)
```

Comment:
Will double check for `str` but I mostly copy/pasted the way Django does with `ugettext_lazy` so it should be fine.